Library/Library/AddLibrary.aspx

```diff
@@ -0,0 +1,41 @@
+﻿<%@ Page Title="" Language="C#" MasterPageFile="~/Site.Master" AutoEventWireup="true" CodeBehind="AddLibrary.aspx.cs" Inherits="Library.AddLibrary" %>
+
+<asp:Content ID="Content5" ContentPlaceHolderID="main" runat="server">
+     <h2>Add Library</h2>
+
+    <fieldset>
+        <div>
+            <asp:label id="LibNameLabel" runat="server"  text="Name: " />
+            <asp:textbox id="LibName" runat="server" />
+        </div>
+        <div>
+            <asp:label id="AddressLabel" runat="server"  text="Address: " />
+            <asp:textbox id="AddressInput" runat="server" />
+        </div>
+
+        <div>
+           <asp:label id="City" runat="server"  text="City: " />
+            <asp:textbox id="CityInput" runat="server" />
+        </div>
```

Comment:
Whatever your convention is, work towards consistency with your control `ID` naming. For example, every label will end with the suffix "Label" and every input control will end with the suffix of "Input".